svelte/fr/tutorial/01-introduction/07-making-an-app/text.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Créer une application
+---
+
+Ce tutoriel est construit pour que vous vous familiarisiez avec le processus d'écriture de composants. Mais au bout d'un moment, vous aurez envie d'écrire vos composants dans le confort de votre éditeur de texte préféré.
+
+Tout d'abord, vous aurez besoin d'un outil de compilation pour vous servir de Svelte. Nous recommandons d'utiliser [SvelteKit](https://kit.svelte.dev), qui met en place [Vite](https://vitejs.dev/) et [vite-plugin-svelte](https://github.com/sveltejs/vite-plugin-svelte/) pour vous...
+
+
+```bash
+npm create svelte@latest myapp
+```
+
+Il existe également un certain nombre d'[outils gérés par la communauté](https://sveltesociety.dev/tools).
+
+Si vous êtes relativement débutant•e dans le développement web, ne vous inquiétez pas si vous n'avez jamais utilisé ces outils. Nous avons préparé le guide [Svelte for new developers](/blog/svelte-for-new-developers), qui vous guidera pas-à-pas.
```

Comment:
On va traduire les billets de blog ? Certains auraient presque intérêt à être ajouté dans le doc...
Je distingue les articles de blogs peu fréquents (3/4 par an) des nouveautés dans la communauté (1 par mois) que l'on n'a pas intérêt à traduire à mon avis.

Si on ne le traduit pas, je traduirais "Svelte for new developers" en précisant entre parenthèse "en anglais"

Reply:
hhhmmm, ça serait une bonne idée, dans un troisième temps (après avoir traduit la doc de SvelteKit)

En attendant, je garderais le nom des articles en anglais, avec cette nuance de traduction :
```suggestion
Si vous êtes relativement débutant•e dans le développement web, ne vous inquiétez pas si vous n'avez jamais utilisé ces outils. Il existe le guide [Svelte for new developers](/blog/svelte-for-new-developers) (en anglais), qui vous guidera pas-à-pas.
```

Reply:
```suggestion
Si vous êtes relativement débutant•e dans le développement web, ne vous inquiétez pas si vous n'avez jamais utilisé ces outils. Vous pouvez suivre le guide [Svelte for new developers](/blog/svelte-for-new-developers) (en anglais), qui vous guidera pas-à-pas.
```